options/linux/generic/sys-klog.cpp

```diff
@@ -0,0 +1,16 @@
+#include <errno.h>
+#include <sys/klog.h>
+
+#include <bits/ensure.h>
+
+#include "mlibc/linux-sysdeps.hpp"
```

Comment:
Why the use of `”` here? We should be able to use `<>` here if I’m not mistaken.